registry/storage/garbagecollect_test.go

```diff
@@ -23,11 +22,7 @@ type image struct {
 
 func createRegistry(t *testing.T, driver driver.StorageDriver, options ...RegistryOption) distribution.Namespace {
 	ctx := context.Background()
-	k, err := libtrust.GenerateECP256PrivateKey()
-	if err != nil {
-		t.Fatal(err)
-	}
-	options = append([]RegistryOption{EnableDelete, Schema1SigningKey(k), EnableSchema1}, options...)
+	options = append([]RegistryOption{EnableDelete}, options...)
```

Comment:
Let's make this more idiomatic `Go`:

```suggestion
	options = append(options, EnableDelete)
```